tags/resources/toc-supporting-guides/tech-papers.md

```diff
@@ -0,0 +1,155 @@
+
+### TAG leader sponsorship and cross-TAG awareness
+
+A TAG leader (co-chair or tech lead) must step forward to sponsor the paper. Other TAG leaders may serve as the lead for paper development but the role of sponsor must be independent from the lead for the paper to reduce any bias in the process. The paper's sponsor is responsible for ensuring the development of the paper adheres to this process, is still progressing towards publication, and provides the initial review of the draft before public comment period may open.
+
+The paper sponsor is also expected to coordinate awareness of the paper development and timelines to other TAGs to ensure they are appropriately included and have the opportunity to contribute and participate.
+
+### Refining the scope, audience and participants
+
+Interest in participating is expressed on the Issue by comments of those individuals. Soliciting interest and participation must allow for sufficient time (no less than two weeks) for fair opportunity to gather interest. Interested parties should then meet to define timelines, refine the scope, declare the audience, and determine paper lead. It is recommended that TAGs leverage communication channels to increase awareness of the proposed paper, its topic, and to solict potential contributors.
+
+## Managing paper development
+
+Once the group has met to define timelines, scope, audience, and determined the lead, the Issue must be updated to provide the following information (initial estimates provided but may be adjusted by the group or TAG leadership as appropriate):
+
+| Milestone | Estimated Time |
+| --- | --- |
+| Audience, Goals, & refining scope | 1 week |
+| Outline | 2 weeks |
+| Tasking Assignment | 1 week |
+| Content Rough-in | 2-3 weeks |
+| Collaborative Review | 2 weeks |
+| Executive Summary and content wrap up | 2 weeks |
+| Narrative Voice | 1-2 weeks |
+| Final Group Review | 1 week |
+| Community Review | 2 weeks |
+| Public comment adjudication | 2 weeks (simultaneous with review) |
+| Leadership Review | 1-2 weeks |
+| CNCF publishing engagement | ~2-3 weeks |
+| Addition to the repo | 2 weeks |
+| Blog post and publishing coordination | 2-3 weeks |
+
+### Milestones Details
+
+#### Audience, Goals, and Refining Scope
+
+Define the following:
+
+- Who is the intended audience?
+- What positions do they hold?
+- What problem are we addressing?
+- What assumptions are we making about the audience or content?
+- Document these in the "Introduction > Assumptions" section.
+
+#### Outline
+
+The outline needs crafted and placed in a shared document (google docs, hackmd, or other collaborative documentation tool). It will serve as the basis for paper structure and tasking. The Issue must be updated to link to the shared document.
```

Comment:
```suggestion
The outline needs to be crafted and placed in a shared document (google docs, hackmd, or other collaborative documentation tool). It will serve as the basis for paper structure and tasking. The Issue must be updated to link to the shared document.
```